doc/conf.py

```diff
@@ -0,0 +1,146 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
```

Comment:
I _think_ for autosummary we need to have
```suggestion
import sys
sys.path.append(os.path.abspath(".."))
```

Reply:
Turns out this isn't needed :) Not sure why, but it seems to work.